### DISCUSSION TITLE
Change Camera Refresh time frame: temp vs permanent?

### DISCUSSION THREAD

**Discussion by dwdonnelly** [mrlt8/docker-wyze-bridge#1285] [Q&A] [answered]
@mrlt8 On the WEB UI, there appears to be the ability to change the refresh rate:
![image](https://github.com/mrlt8/docker-wyze-bridge/assets/42418057/b1898894-d2a2-4abe-81b0-38c509fcc6e3)

When I go to the configuration page, there is the Snapshot Mode selection refresh rate.

What is the difference between the two settings?

Is the Configuration page Snapshot setting what one desires on a permanent basis (until you change it) and the WEB UI 
![image](https://github.com/mrlt8/docker-wyze-bridge/assets/42418057/b1898894-d2a2-4abe-81b0-38c509fcc6e3)
value is a temporary change-ability/feature?

Thanks in advance for your help.

**Comment by mrlt8** [comment, marked as answer]
The refresh value in the WebUI is for updating the snapshots while the WebUI is open while the one in the config will continue to update at the set interval.

Keep in mind that snapshots are pretty CPU intensive, so it may be more efficient to just view the streams directly.

**Reply by dwdonnelly** [reply]
@mrlt8 Many thanks.  Things are slowly coming into perspective.

Thanks again for your quick responses - they are greatly appreciated.

And again - thanks for a most excellent integration !!
